README.md

```diff
@@ -3,6 +3,7 @@
 [![License: GPL v3](https://img.shields.io/badge/License-GPLv3-blue.svg)](https://www.gnu.org/licenses/gpl-3.0)
 [![GitHub stars](https://img.shields.io/github/stars/PalisadoesFoundation/talawa-api.svg?style=social&label=Star&maxAge=2592000)](https://github.com/PalisadoesFoundation/talawa-api)
 [![GitHub forks](https://img.shields.io/github/forks/PalisadoesFoundation/talawa-api.svg?style=social&label=Fork&maxAge=2592000)](https://github.com/PalisadoesFoundation/talawa-api)
+[![codecov](https://codecov.io/gh/sandeep-v1404/talawa-api/branch/test-coverage-check/graph/badge.svg?token=LZ0LY3XQJ8)](https://codecov.io/gh/sandeep-v1404/talawa-api)
```

Comment:
This needs to be changed to our badge

[![codecov](https://codecov.io/gh/PalisadoesFoundation/talawa-admin/branch/develop/graph/badge.svg?token=II0R0RREES)](https://codecov.io/gh/PalisadoesFoundation/talawa-admin)

Reply:
Sir, this needs the authorization to get the badge code.
![image](https://user-images.githubusercontent.com/57038543/148376560-eebd542d-d006-4e52-97c2-dd8ee3980c3a.png)

Reply:
I have already registered on behalf of the organization. Please use the code provided

Reply:
Yes 

> I have already registered on behalf of the organization. Please use the code provided

But the code is related to `talawa-admin`